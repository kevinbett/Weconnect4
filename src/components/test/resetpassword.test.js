import ResetPassword from '../ResetPasswordForm'
import { shallow } from "enzyme"

describe('registration page renders',() => { 
    it ('populates the signup form', () => { 
        expect(shallow(<ResetPassword/>).find("form").exists()).toBe(true);
    })
} )
