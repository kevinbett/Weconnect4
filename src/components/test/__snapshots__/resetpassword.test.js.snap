// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`registration page renders populates the signup form 1`] = `
<div
  id="main-registration-container"
>
  <div
    id="register"
  >
    <h3>
      Enter email to reset
    </h3>
    <form
      onSubmit={[Function]}
    >
      <p
        className="errorMsg"
      />
      <label>
        Email ID:
      </label>
      <input
        name="emailid"
        type="text"
      />
      <input
        className="button"
        type="submit"
        value="Submit"
      />
    </form>
  </div>
</div>
`;
